modules/apps/app-builder/app-builder-test/src/testFunctional/macros/App.macro

```diff
@@ -18,6 +18,23 @@ definition {
 		Button.click(button = "Deploy");
 	}
 
+	macro clickDeploy {
+		Button.click(button = "Deploy");
+	}
+
+	macro clickNext {
+		Button.click(button = "Next");
+	}
+
+	macro clickPrevious {
+		Button.click(button = "Previous");
+	}
+
+	macro deployAppToggleSwitcher {
+		Check.checkToggleSwitch(
+			key_deployOption = "${deployOption}",
+			locator1 = "AppBuilder#DEPLOY_APP_TOGGLE");
+
```

Comment:
Hi @tainguyen100 , the script is okay, but I noticed that there is a problem at App.macro. This macro deployAppToggleSwitcher is missing a brace } to close the macro. Poshi validation is failing due to this.